samples/sample_common/src/vaapi_utils_x11.cpp

```diff
@@ -64,11 +63,11 @@ X11LibVA::X11LibVA(void)
         }
         if (MFX_ERR_NONE == sts)
         {
-            if (!setenv("LIBVA_DRIVER_NAME", PROCESSING_DRIVER_NAME, 1)) {
-            va_res = m_libva.vaInitialize(m_va_dpy, &major_version, &minor_version);
-            sts = va_to_mfx_status(va_res);
-            if (MFX_ERR_NONE != sts) {
-                m_x11lib.XCloseDisplay(m_display);
+            if (!setenv("LIBVA_DRIVER_NAME", DRIVER_NAME, 1)) {
```

Comment:
nope, you don't need to set _any_ driver name. If needed user will do that in a shell. Just call vaInitialize().